content/en/blog/2022-05-04-april-update.md

```diff
@@ -0,0 +1,607 @@
+self-heal (e.g. retrying failing SSH connections).
+
+#### Experimental managed transport for libgit2 Git implementation
+
+Now has self-healing capabilities, to recover from failure when
+long-running connections become stale.
+
+#### SOPS refactored and optimized
+
+Including various improvements and extended code coverage. Age
+identities are now imported once and reused multiple times, optimizing
+CPU and memory usage between decryption operations.
+
+#### Helm chart directory loader improvements
+
+Introduction of a secure directory loader which improves the handling of
+Helm charts paths.
+
+For a more detailed list of changes in the series, please refer to the
+change logs of
+[0.29.0](https://github.com/fluxcd/flux2/releases/tag/v0.29.0),
+[0.29.1](https://github.com/fluxcd/flux2/releases/tag/v0.29.1),
+[0.29.2](https://github.com/fluxcd/flux2/releases/tag/v0.29.2),
+[0.29.3](https://github.com/fluxcd/flux2/releases/tag/v0.29.3),
+[0.29.4](https://github.com/fluxcd/flux2/releases/tag/v0.29.4),
+and
+[0.29.5](https://github.com/fluxcd/flux2/releases/tag/v0.29.5).
+
+### Flagger 1.20.0
+
+This release comes with improvements to the AppMesh, Contour and Istio
+integrations.
+
+#### Improvements
+
+- AppMesh: Add annotation to enable Envoy access logs
+  [\#1156](https://github.com/fluxcd/flagger/pull/1156)
+- Contour: Update the httproxy API and enable `RetryOn`
+  [\#1164](https://github.com/fluxcd/flagger/pull/1164)
+- Istio: Add destination port when port discovery and delegation are
+  true
+  [\#1145](https://github.com/fluxcd/flagger/pull/1145)
+- Metrics: Add canary analysis result as Prometheus metrics
+  [\#1148](https://github.com/fluxcd/flagger/pull/1148)
+
+#### Fixes
+
+- Fix canary rollback behaviour
+  [\#1171](https://github.com/fluxcd/flagger/pull/1171)
+- Shorten the metric analysis cycle after confirm promotion gate is
+  open
+  [\#1139](https://github.com/fluxcd/flagger/pull/1139)
+- Fix unit of time in the Istio Grafana dashboard
+  [\#1162](https://github.com/fluxcd/flagger/pull/1162)
+- Fix the service toggle condition in the podinfo helm chart
+  [\#1146](https://github.com/fluxcd/flagger/pull/1146)
+
+### Flux Ecosystem
+
+#### Flux Subsystem for Argo
+
+In the latest release, we have added checkboxes to enable Flux Subsystem
+in the Argo CD UI. We also have a [tutorial to use TF-controller with
+Flux Subsystem for
+Argo](https://flux-subsystem-argo.github.io/website/tutorials/terraform/).
+With this you have an alternative option to Crossplane to manage
+infrastructure.
+
+#### Terraform-controller
+
+We have released TF-controller v0.9.4 which is a bug fixed release. We
+also added **cloud cost estimation** to our road map. Please feel free
+to give us feedback on how you would like this feature to be:
+
+- Issues here: <https://github.com/weaveworks/tf-controller/issues>, and
+- Discussions here: <https://github.com/weaveworks/tf-controller/discussions>
+
+#### Weave GitOps
```

Comment:
>Weave GitOps is a powerful, open source extension to Flux, which
provides insights into your deployments, and makes continuous delivery
with GitOps easier to adopt and scale across your teams. 
With the 0.7.0 release you can easily install alongside an existing Flux setup, adding
(or removing) Weave GitOps as a standard Helm resource. The new Web UI
provides deep insights into Flux resources to enable you to understand
their relationships and uncover their statuses.

Updating to reflect v0.8.0 release:  

Weave GitOps is a powerful, open source extension to Flux, which
provides insights into your deployments, and makes continuous delivery
with GitOps easier to adopt and scale across your teams. You can easily install alongside an existing Flux setup, adding (or removing) Weave GitOps as a standard Helm resource. 

The v0.8.0 release brings multi-namespace querying so you can see objects from across your cluster in the Web UI, several UI enhancements and bug fixes, as well as supporting the Source v1beta2 API - this breaking change means we now require Flux v0.29.0 or later.

Reply:
Super - added. Thanks @sympatheticmoose!

Reply:
https://deploy-preview-911--fluxcd.netlify.app/blog/2022/05/april-2022-update/#weave-gitops